doc/port_auto_neg/port-auto-negotiation-design.md

```diff
@@ -603,7 +672,7 @@ I choose this solution because:
 
 Dynamic port breakout feature also introduces a hwsku.json file to describe the port capability. It defines the default dynamic breakout mode for now. As we won't automatically set auto negotiation attributes for a port after port breakout, it is not necessary to change the hwsku.json in this feature.
 
-#### xcvrd Consideration
+#### PMON xcvrd Consideration
```

Comment:
Could you provide an example of media_settings.json for the new attributes?

Reply:
These capabilities are directly derived from the transceiver info, and yes, we could provide the custom overrides in the media_settings.json, will update the HLD